layouts/partials/download/download.html

```diff
@@ -2,8 +2,8 @@ <h3>Get O3DE [Developer Preview]</h3>
 <div class="download_release">Release 2107.1</div>
```

Comment:
Missing a class for the "system requirements" to also be toggled along with the download button and text.